test/ig/IGExporter.IG.test.ts

```diff
@@ -3347,6 +3371,10 @@ describe('IGExporter', () => {
           }
         ]
       };
+      config.menu = [
+        { name: 'Home', url: 'index.html' },
+        { name: 'Artifacts', url: 'artifacts.html' }
+      ];
```

Comment:
This is super nit-picky, but given that you define the whole config JSON in the lines just above this, I think it makes more sense just to stick the menu in that JSON object in the first place.